server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/event/ComputeMessageFastViewProjectionListener.java

```diff
@@ -95,11 +91,6 @@ private void handleAddedEvent(Added addedEvent, MailboxSession session) throws M
 
     @VisibleForTesting
     MessageFastViewPrecomputedProperties computeFastViewPrecomputedProperties(MessageResult messageResult) throws MailboxException, IOException {
-        MessageFullView message = messageFullViewFactory.fromMessageResults(ImmutableList.of(messageResult));
-
-        return MessageFastViewPrecomputedProperties.builder()
-            .preview(Preview.from(message.getPreview().getValue()))
-            .hasAttachment(message.isHasAttachment())
-            .build();
+        return messageFastViewPrecomputedPropertiesFactory.from(messageResult);
```

Comment:
Not relying on `messageFullViewFactory.fromMessageResults` allows to be purely CPU bound (mime parsing over a byte array) hence no issue scheduling this on the parrallel scheduler...

(That is why I cherry-picked this comment)